packages/maskbook/src/plugins/ITO/UI/ClaimDialog.tsx

```diff
@@ -256,8 +259,11 @@ export function ClaimDialog(props: ClaimDialogProps) {
                         value === ''
                             ? new BigNumber(0)
                             : new BigNumber(value).multipliedBy(new BigNumber(10).pow(claimToken.decimals))
-                    setClaimAmount(val)
-                    setTokenAmount(val.dividedBy(ratio))
+                    const isMax = value === formatBalance(new BigNumber(maxAmount), claimToken.decimals)
```

Comment:
Fix with current `<TokenAmountPanel />`, we will lose some precision which lead to 

A. click max button or input max number

<img src="https://user-images.githubusercontent.com/63733714/107138186-f6dcdd00-694d-11eb-8fdb-c002a4ea9b9c.png" width=400 />

B. using slider

<img src="https://user-images.githubusercontent.com/63733714/107138224-2b509900-694e-11eb-8140-e470f4c3f879.png" width=400 />

Reply:
I noticed that `dp(0)` will use the configured `ROUNDING_MODE` by default. And in this case, `79388027.63429488764` seems rounded to `79388028` which becomes a larger number. I don't know if this is a conversion way for a finance-related app? @xiazhiming Any advice?

Reply:
Notice that MSKD is 6 decimals, so 79.388028 (79388028) is right, there's no `79388027.63429488764` for MSKD. So if there's no `isMax` handling, you can't get all 0.60142445177496127 MSKA from `<TokenAmountPanel />`, the largest MSKA you can get is to input 79.388027. 79.388028 * raito would greater than 0.60142445177496127